### PR TITLE
PR for pre-1.0 release 

### PR DESCRIPTION
**Pull request by jd-lara**
This PR is meant to review the current branch for a pre 1.0 release.

### CODE REVIEW

File: UC_test/ramp_test.json
Comment (daniel-thom) [inline UC_test/ramp_test.json:1]: I don't know how these are used but they won't work any more.

**Review by daniel-thom — COMMENTED** [review]
This looks fine. Just need to either update the serialized JSON files or delete them.

File: UC_test/ramp_test.json
Comment (jd-lara) [reply UC_test/ramp_test.json:1]: These are old and can be removed

**Review by daniel-thom — APPROVED** [review]

**Comment by jd-lara** [comment]
Will merge once all other pre-10 branches are approved too.

**Comment by jd-lara** [comment]
I am merging this PR to make the tests pass by default in the packages

**Pull request merged**
